runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -1517,11 +1517,9 @@ MM_ObjectAccessBarrier::copyObjectFields(J9VMThread *vmThread, J9Class *objectCl
 		/* initialize lockword, if present */
 		lockwordAddress = getLockwordAddress(vmThread, destObject);
 		if (NULL != lockwordAddress) {
-			if (J9_ARE_ANY_BITS_SET(J9CLASS_EXTENDED_FLAGS(objectClass), J9ClassReservableLockWordInit)) {
-				J9_STORE_LOCKWORD(vmThread, lockwordAddress, OBJECT_HEADER_LOCK_RESERVED);
-			} else {
-				J9_STORE_LOCKWORD(vmThread, lockwordAddress, 0);
-			}
+			j9objectmonitor_t lwValue = VM_ObjectMonitor::getInitialLockword(vmThread->javaVM, objectClass);
+
```

Comment:
Would you please remove this empty line?

Reply:
I'll remove it.